documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -2699,14 +2713,21 @@ You can configure the connector to skip the following types of operations:
 
 By default, no operations are skipped.
 
-|[[oracle-property-signal-data-collection]]<<oracle-property-signal-data-collection, `+signal.data.collection+`>>
-|No default
-| Fully-qualified name of the data collection that is used to send {link-prefix}:{link-signalling}[signals] to the connector.
-The name format is _database_name.schema-name.table-name_.
+
+|[[oracle-property-signal-data-collection]]<<oracle-property-signal-data-collection,`+signal.data.collection+`>>
+|No default value
+a|Fully-qualified name of the data collection that is used to send {link-prefix}:{link-signalling}[signals] to the connector. +
+Use the following format to specify the collection name: +
+`_<schemaName>_._<tableName>_`
```

Comment:
Please be careful here - and check other instances too that you keep the name format as now it would be broken.

Reply:
@jpechane Can you clarify? I'm not sure what part is incorrect. Is it the formatting, or should the text specify something other than `schemaName.tableName`?

Reply:
@roldanbob PLease see the original text was saying the format is database_name.schema-name.table-name so three components, not two.
This is different for each connector so when you are chnaging this part please make sure you don't change the format of the name.